regulations/views/partial_sxs.py

```diff
@@ -89,9 +89,10 @@ def get_context_data(self, **kwargs):
         label_id = context['label_id']
         part = label_id.split('-')[0]
         notice_id = context['notice_id']
+        version = context['version']
```

Comment:
Are we sure context will always have `version`? Would it be safer to use `context.get('version', notice_id)`, or would that be redundant (will the context always have `notice_id`)?